### PR TITLE
feat(content): Added wooden flute

### PR DESCRIPTION
**Pull request by yay855**
Added a wooden equivalent of the bone flute, since wood flutes... you know, are a thing.

## Purpose of change

The bone flute was a good earlygame musical instrument, but _bone_ flutes are _not_ the most common type- wooden ones are.

## Describe the solution

Added in a wooden flute, crafting recipes, and appropriate spawns.

### CODE REVIEW

File: data/json/items/tool/musical_instruments.json [hunk 1]
```diff
@@ -61,6 +61,28 @@
...
+      "volume": 12,
+      "fun": -5,
+      "fun_bonus": 2,
+      "speed_penalty": 10,
+      "description_frequency": 20,
+      "player_descriptions": [
+        "You play a little tune on your flute.",
+        "You play a beautiful piece on your flute.",
+        "You play a piece on your flute that resembles the howling wind through Aurignacian caves.",
+        "Your bone flute wails for the fallen during the hunt."
```
Comment (chaosvolt) [inline data/json/items/tool/musical_instruments.json:82]:
```suggestion
        "Your flute wails for the fallen during the hunt."
```
Unfortunate that the lines are all the same as the bone one but oh well.

File: data/json/items/tool/musical_instruments.json [hunk 2]
```diff
@@ -61,6 +61,28 @@
       ]
     }
   },
+  {
+    "id": "wood_flute",
+    "type": "TOOL",
+    "name": { "str": "wooden flute" },
+    "copy-from": "bone_flute",
```
Comment (chaosvolt) [inline data/json/items/tool/musical_instruments.json:68]: Also needs a description change.
```suggestion
    "copy-from": "bone_flute",
    "description": "A simple flute carved from wood with five finger holes.",
```

File: data/json/recipes/other/tools.json [hunk 3]
```diff
@@ -410,6 +410,18 @@
...
     "components": [ [ [ "bone_sturdy", 1, "LIST" ] ] ]
   },
+  {
+    "result": "wood_flute",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "1 h",
```
Comment (VissValdyr) [inline data/json/recipes/other/tools.json:420]:
```suggestion
    "time": "40 m",
```

File: data/json/recipes/other/tools.json [hunk 4]
```diff
@@ -410,6 +410,18 @@
...
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
```
Comment (VissValdyr) [inline data/json/recipes/other/tools.json:422]:
```suggestion
    "qualities": [ { "id": "DRILL", "level": 1 } ],
```

File: data/json/recipes/other/tools.json [hunk 5]
```diff
@@ -410,6 +410,18 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "bone_sturdy", 1, "LIST" ] ] ]
   },
+  {
+    "result": "wood_flute",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
```
Comment (VissValdyr) [inline data/json/recipes/other/tools.json:419]:
```suggestion
    "difficulty": 2,
```

**Comment by VissValdyr** [comment]
Windows doesn't like you somehow

**Comment by chaosvolt** [comment]
image

Wuaaah!?

**Review by chaosvolt — APPROVED** [review]
Aight, re-running failed jobs fixed it so no idea, weird.

**Pull request merged**